exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlptrace.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
+
+To constructs a new OTLP trace Exporter, you can follow this 
+ [`example`](https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/otlp/otlptrace/example_test.go).
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlptrace
+```
+
+## [`otlptrace`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+The `otlptrace` package provides an exporter implementing the OTel span exporter interface.
+This exporter is configured using a client satisfying the `otlptrace.Client` interface.
+This client handles the transformation of data into wire format and the transmission of that data to the collector.
+
+## [`otlptracegrpc`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc)
+
+The `otlptracegrpc` package implements a gRPC client to be used in the span exporter.
+
+##  [`otlptracehttp`](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp)
+
+The `otlptracehttp` package implements a HTTP client to be used in the span exporter.
```

Comment:
The [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#specify-protocol) mentions two kinds of HTTP protocols. Which one is it? Both and it's configurable or just one of them?

Reply:
Found in godoc:

> Package otlptracehttp a client that sends traces to the collector using HTTP with binary protobuf payloads.

Reply:
According to your description,I think the specify protocol is http/protobuf.
Am I wrong?

Reply:
Do I need to add the specified protocol to README?

Reply:
Yes. Please add it.

Reply:
Updated, PTAL.